api/src/lib/db.ts

```diff
@@ -64,6 +64,8 @@ export async function getPrismaClient() {
   return client
 }
 
+export default getPrismaClient
```

Comment:
I did this at first when attempting to use `globalSetup` (which won't work for tests), but left it in to simplify the setup code. Will remove if confusing.